docs/UserGuide.md

```diff
@@ -59,17 +59,18 @@ StudentConnect is a solution for hassle-free team formation for students to brow
 5. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
    Some example commands you can try:
 
-    * `list` : Lists all students.
-
-    * `add n/John Doe m/Computer Science y/2 e/johnd@u.nus.edu d/I love programming in my free time t/02 17 20 sm/https://www.linkedin.com/in/john-doe-123456789 nt/local g/m` : Adds a student named `John Doe` to the StudentConnect system.
-
-    * `delete [email]` : Deletes the student with the corresponding email.
-
-    * `clear` : Deletes all data from the system (i.e. students, groups, and tasks).
-
-    * `exit` : Exits the app.
-
-    * `create` : Creates a new group.
+    * `list` : 
+        * Lists all students in the StudentConnect system.
+    * `add n/John Doe m/Computer Science y/2 e/johnd@u.nus.edu d/I love programming in my free time t/02 t/17 t/20 sm/https://www.linkedin.com/in/john-doe-123456789 nt/local g/m` : 
+        * Adds a student named `John Doe` and their details to the StudentConnect system.
+    * `delete [email]` : 
```

Comment:
should we change this to `delete EMAIL` since this is the given format below

Reply:
okay!